docs/guides/guided_generation.md

```diff
@@ -0,0 +1,67 @@
+# Guided Generation
```

Comment:
Please add this to navigation in `mkdocs.yml`

Reply:
Whoops! Fixed.